android/src/main/java/com/reactlibrary/securekeystore/RNSecureKeyStorePackage.java

```diff
@@ -13,16 +13,22 @@
 import com.facebook.react.ReactPackage;
 import com.facebook.react.bridge.NativeModule;
 import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.JavaScriptModule;
 import com.facebook.react.uimanager.ViewManager;
 
 public class RNSecureKeyStorePackage implements ReactPackage {
-  @Override
-  public List<NativeModule> createNativeModules(ReactApplicationContext reactContext) {
-    return Arrays.<NativeModule>asList(new RNSecureKeyStoreModule(reactContext));
-  }
+    @Override
+    public List<NativeModule> createNativeModules(ReactApplicationContext reactContext) {
+        return Arrays.<NativeModule>asList(new RNSecureKeyStoreModule(reactContext));
+    }
 
-  @Override
-  public List<ViewManager> createViewManagers(ReactApplicationContext reactContext) {
-    return Collections.emptyList();
-  }
+    @Override
+    public List<Class<? extends JavaScriptModule>> createJSModules() {
```

Comment:
This was removed because the latest version of react-native is not supporting this. Which version of react-native are you using?